crates/sui-indexer/src/bin/indexer_data_validation.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cmp::max;
+
+use anyhow::Result;
+use clap::Parser;
+use tracing::{error, info, warn};
+
+use sui_indexer::new_rpc_client;
+use sui_json_rpc_types::{
+    CheckpointId, ObjectChange, SuiObjectDataOptions, SuiTransactionResponseOptions,
+};
+use sui_sdk::SuiClient;
+use sui_types::base_types::{ObjectID, SequenceNumber};
+use sui_types::digests::TransactionDigest;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    info!("Running correctness check for indexer...");
```

Comment:
Need to add this for the log to show up in terminal

```suggestion
async fn main() -> Result<()> {
    // Initialize logging
    let (_guard, filter_handle) = telemetry_subscribers::TelemetryConfig::new()
      .with_env()
      .with_prom_registry(&prometheus_registry)
      .init();
    info!("Running correctness check for indexer...");
```